dev/com.ibm.ws.microprofile.faulttolerance.cdi.2.1/bnd.bnd

```diff
@@ -0,0 +1,49 @@
+#*******************************************************************************
```

Comment:
Please change the naming of these projects so that it follows similar conventions to MP Config.
so this one would be `com.ibm.ws.microprofile.faulttolerance.2.1.cdi`. This makes it clear that it is a FT 2.1 project and not an integration with CDI 2.1